Dfe.Academies.Academisation.Data.UnitTest/ProjectAggregate/ProjectStatusesDataQueryTests.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using AutoFixture;
+using Dfe.Academies.Academisation.Data.ProjectAggregate;
+using Dfe.Academies.Academisation.Data.UnitTest.Contexts;
+using Xunit;
+
+namespace Dfe.Academies.Academisation.Data.UnitTest.ProjectAggregate
+{
+	public class ProjectStatusesDataQueryTests
+	{
+		private readonly Fixture _fixture = new();
+
+		private readonly ProjectStatusesDataQuery _subject;
+		private readonly AcademisationContext _context;
+
+		public ProjectStatusesDataQueryTests()
+		{
+			_context = new TestProjectContext().CreateContext();
+			_subject = new ProjectStatusesDataQuery(_context);
+		}
+
+		[Fact]
+		public async Task ProjectsExist__DuplicateStatuses__ReturnsDistinct()
+		{
+			var status = "Active";
+			_context.Projects.Add(new ProjectState() { ProjectStatus = status });
+			_context.Projects.Add(new ProjectState() { ProjectStatus = status });
+			_context.Projects.Add(new ProjectState() { ProjectStatus = status });
+
+			await _context.SaveChangesAsync();
+
+			var results = await _subject.Execute();
+
+			Assert.Multiple(
+				() => Assert.Equal(status, results.First()),
+				() => Assert.Single(results)
+			);
+		}
+
+		[Fact]
```

Comment:
@dneed-nimble I did one of those here. 
Though, happy to put another one in.

Reply:
Happy to resolve